template/template.go

```diff
@@ -133,6 +133,24 @@ func (p *PostProcessor) MarshalJSON() ([]byte, error) {
 	return json.Marshal(m)
 }
 
+// If Post Processor Is not in this list it means that it's valid for all builders
+var validBuildersPerPostProcessor = map[string][]string{
+	"vsphere":          {"vmware-iso", "vmware-vmx"},
+	"vsphere-template": {"vmware-iso", "vmware-vmx"},
+}
+
+func (p *PostProcessor) IsValidToBuilder(builder string) bool {
+	if builders, ok := validBuildersPerPostProcessor[p.Name]; ok {
```

Comment:
NIT: do you think it would be a good idea to make of that if a guard clause ? For example:
```go
builders, ok := validBuildersPerPostProcessor[p.Name]
if !ok {
    return true
}
```

I personally think that guard clauses help a lot with readability.

Reply:
I also think it looks better the way you suggesting!